tools/project-context.js

```diff
@@ -596,10 +597,21 @@ _.extend(ProjectContext.prototype, {
     var self = this;
     var searchDirs = [files.pathJoin(self._projectDirForLocalPackages, 'packages')];
 
-    if (! self._ignorePackageDirsEnvVar && process.env.PACKAGE_DIRS) {
-      // User can provide additional package directories to search in
-      // PACKAGE_DIRS (colon-separated).
-      _.each(process.env.PACKAGE_DIRS.split(':'), function (p) {
+    // User can provide additional package directories to search in
+    // METEOR_PACKAGE_DIRS (colon-separated),
+    // This was added after 1.4.0.1
+    var envPackageDirs = process.env.METEOR_PACKAGE_DIRS || process.env.PACKAGE_DIRS;
```

Comment:
Per the original feature request, one of the original requirements was for:

> **The existing PACKAGE_DIRS variable should keep working exactly as it is right now.**

...this was because there over 100 pages of GitHub search results of projects using `PACAKGE_DIRS` already.

By doing it this way (e.g. `METEOR_PACKAGE_DIRS || PACKAGE_DIRS`) and changing the code below to use `path.delimiter`, this requirement is no longer met which means this becomes a breaking-change for Windows projects which are using `PACKAGE_DIRS`.

Therefore, I think it's important that`PACKAGE_DIRS` continue to use `:` regardless of the OS since this is the way it has behaved in the past and it's used quite frequently.